images/dind/docker.service

```diff
@@ -1,15 +0,0 @@
-[Unit]
-Description=Docker-in-Docker (dind)
-After=network.target
```

Comment:
This dependency is retained.  A drop-in unit file overrides settings in the distro or host service file, and this line does appear in the default docker.service.